CefSharp.OutOfProcess.BrowserProcess/CallbackProxies/DialogHandlerProxy.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using CefSharp.OutOfProcess.Interface;
+using CefSharp.Handler;
+using System.Linq;
+using CefSharp.OutOfProcess.Interface.Callbacks;
+
+namespace CefSharp.OutOfProcess.BrowserProcess.CallbackProxies
+{
+
+    internal sealed class DialogHandlerProxy : CallbackProxyBase<IFileDialogCallback>, IDialogHandler
+    {
+        public DialogHandlerProxy(IOutOfProcessHostRpc host)
+            : base(host)
+        {
+
+        }
+
+        public void Callback(FileDialogCallbackDetails details)
```

Comment:
add argument check for details == null  and details.Files == null

Reply:
done